api/types/src/convert.rs

```diff
@@ -297,47 +323,52 @@ impl<'a, R: MoveResolverExt + ?Sized> MoveConverter<'a, R> {
         }
     }
 
-    // TODO: Consider expanding ResourceGroup into Resource and returning Vec<WriteSetChange>
-    pub fn try_access_path_into_write_set_change(
+    pub fn try_access_path_into_write_set_changes(
         &self,
         state_key_hash: String,
         access_path: AccessPath,
         op: WriteOp,
-    ) -> Result<WriteSetChange> {
+    ) -> Result<Vec<WriteSetChange>> {
```

Comment:
Isn't this a breaking change?

Reply:
Results get flattened later where this is called.